accounts/src/accounts.rs

```diff
@@ -282,10 +290,11 @@ impl Accounts {
         Ok(receipt)
     }
 
-    fn prepare_receipts(receipts: &Receipts) -> (ReceiptsMap, ReceiptsMap, ReceiptsMap, Vec<&PrunedAccount>) {
+    fn prepare_receipts(receipts: &Receipts) -> (ReceiptsMap, ReceiptsMap, ReceiptsMap, ReceiptsMap, Vec<&PrunedAccount>) {
```

Comment:
Maybe we should make a struct type that we return here instead of that tuple. But that's only a suggestions... one that Clippy is likely to make :wink: